lib/internal/url.js

```diff
@@ -8,6 +8,8 @@ const {
 const binding = process.binding('url');
 const context = Symbol('context');
 const cannotBeBase = Symbol('cannot-be-base');
+const cannotHaveUsernamePasswordPort =
+    Symbol('cannot have a username/password/port');
```

Comment:
Perhaps `Symbol('cannot-have-username-password-port')` to remain consistent with the other symbols?